backend/sozisel/lib/sozisel_web/schema/types/event_result_types.ex

```diff
@@ -57,11 +59,12 @@ defmodule SoziselWeb.Schema.Types.EventResultTypes do
   end
 
   union :presenter_event_result_data do
-    types [:quiz_simple_result, :poll_result]
+    types [:quiz_simple_result, :poll_result, :whiteboard_result]
 
     resolve_type fn
       %{total_points: _}, _ -> :quiz_simple_result
       %PollResult{}, _ -> :poll_result
+      %{path: _, text: _}, _ -> :whiteboard_result
```

Comment:
Don't you have the `WhiteboardResult` module? the quiz result is a map on purpose as the result types changes, here it probably does not

Reply:
Previously, there was a metadata field, I thought that it would be used for something else in the future. So that I sent to the presenter filtered data without metadata but after removing: metadata I will change it